src/main/java/com/arcbees/plugin/idea/dialogs/ContentSlotDialog.java

```diff
@@ -17,15 +17,15 @@
 package com.arcbees.plugin.idea.dialogs;
 
 import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.module.Module;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.ui.DialogWrapper;
-import com.intellij.psi.JavaPsiFacade;
-import com.intellij.psi.PsiClass;
-import com.intellij.psi.PsiMember;
+import com.intellij.psi.*;
```

Comment:
You can set that up in IntelliJ to wait until you have 999 classes to put the star (which means in practice, never).